clients/studio/ConfigDialog.h

```diff
@@ -120,26 +123,30 @@ namespace Aseba
 		QCheckBox* newCheckbox(QString label, QString ID, bool checked = false);
 
 	protected:
-		template <class T> struct WidgetCache {
+		template<class T>
+		struct WidgetCache
+		{
 			WidgetCache() {}
-			WidgetCache(QWidget* widget, T value):widget(widget),value(value){}
+			WidgetCache(QWidget* widget, T value) :
```

Comment:
Can we configure the style tool to not change the style for constructors? The nicest depends on the context.